src/server/custom-functions/metadata.parser.ts

```diff
@@ -10,7 +10,7 @@ const TYPE_MAPPINGS = {
   [ts.SyntaxKind.BooleanKeyword]: 'boolean',
 };
 
-const CUSTOM_FUNCTION_OPTIONS_KEYS = ['stream', 'volatile'];
+const CUSTOM_FUNCTION_OPTIONS_KEYS = ['cancelable', 'volatile'];
```

Comment:
Do we have support for "volatile" functions?

Reply:
Good catch.  yeah, I don't think the platform does, actually.

Reply:
Actually, @nico-bellante , you might just want to remove both for now.  Either that, or add proper cancelable support (where "proper" means that if a function is declared as customable, I think it also gets the same interface as the streaming one but without setResult, just the oncancelled.. or something like that).  Honestly, I would just drop it for now.